R/summarizeEnvironment.R

```diff
@@ -0,0 +1,37 @@
+# function to generate a data frame with the count of cells of a specified type (e.g. clustered celltypes) in the neighborhood of each cell.
+# ... to be written
+
+summarizeEnvironment <- function(object,
+                                 graphName,
+                                 group,
+                                 name = NULL){
+
+  if (! graphName %in% colPairNames(object)) {
+    stop("graphName is not a graph in the current object")
+  }
+
+  if (! group %in% colnames(colData(object))) {
+    stop("group is not a valid colData entry in the current object")
+  }
+
+  cur_graph <- graph_from_edgelist(as.matrix(colPair(object,graphName)))
+
+  cur_dat <- as_data_frame(cur_graph)
+
+  cur_out <- cur_dat %>%
```

Comment:
In general it is advised to use a much base R as possible to reduce the dependencies of the package and to facilitate testing (external packages change more often than base R). But let's leave it like this until we find something better.